src/raftstore/store/peer.rs

```diff
@@ -1204,7 +1220,16 @@ impl Peer {
                     return Ok(());
                 }
             }
-            ConfChangeType::AddLearnerNode => unimplemented!(),
+            ConfChangeType::AddLearnerNode => {
+                let mut progress = Progress::default();
+                progress.is_learner = true;
+                status.learner_progress.insert(peer.get_id(), progress);
+                return Ok(());
+            }
+            ConfChangeType::PromoteLearnerNode => {
+                status.learner_progress.remove(&peer.get_id());
```

Comment:
Why?

Reply:
We need `learner_progress` because it's used when we report `pending_learners` to pd.